airflow/providers/amazon/aws/hooks/emr.py

```diff
@@ -169,6 +169,15 @@ def add_job_flow_steps(
                 )
         return response["StepIds"]
 
+    def terminate_job_flow(self, job_flow_id: str) -> None:
```

Comment:
We generally try to avoid functions in hooks which just wrap boto3 api. You can call the boto3 api directly from the operator